### PR TITLE
Reduce font size in compact table expanded view

### PR DESCRIPTION
**Pull request by mhuth**
Fixes https://issues.redhat.com/browse/ADVISOR-1814

Sorry I missed this :facepalm: 

Screenshot from 2021-06-23 12-29-11

### CODE REVIEW

**Comment by AllenBW** [comment]
@mhuth Travis requires appeasing

**Comment by mhuth** [comment]
> @mhuth Travis requires appeasing

@AllenBW You're quick!  I thought I could quickly fix that one up before anyone noticed :)

File: src/Messages.js [hunk 1]
```diff
@@ -792,7 +792,7 @@ export default defineMessages({
     id: 'riskOfChangeTextOne',
     description: 'Risk of change text explaining a value one',
     defaultMessage:
-      'The risk of change is <strong>very low</strong>, beacause the change takes very little time to implement and there is minimal impact to system operations.',
+      'The risk of change is <strong>very low</strong>, because the change takes very little time to implement and there is minimal impact to system operations.',
```
Comment (AllenBW) [inline src/Messages.js:795]: Let's do an npm run translations then merge this awesome stuff!

**Review by AllenBW — COMMENTED** [review]
@mhuth eeeeee yer right! I saw nothing!  🙈

File: src/Messages.js (hunk 1 above)
Comment (mhuth) [reply src/Messages.js:795]: Damn it.  I forgot that step :disappointed:

**Review by AllenBW — APPROVED** [review]
❤️ 🙇

**Pull request merged**